webpack/routes.js

```diff
@@ -0,0 +1,565 @@
+/*
```

Comment:
How does this get generated? Do we need to update the docs/PR template to make sure regeneration happens if someone messes with the routes?